uber/models/attendee.py

```diff
@@ -969,7 +998,7 @@ def requested_depts_ids(self, value):
         department_ids = set(
             str(d.department_id) for d in self.dept_membership_requests)
         for department_id in values:
-            if department_id not in department_ids:
+            if str(department_id) not in department_ids:
```

Comment:
This is the root cause of the bug that required the weird deletes in the data migration. The erroneous conditional was comparing `None` to the string `"None"`.